modules/ROOT/pages/_partials/include-key-metrics-apim.adoc

```diff
@@ -0,0 +1,35 @@
+//
+//tag::monitoring-agent-intro[]
+
+Starting October 31, 2024, access to xref:api-manager::analytics-chart.adoc[Key Metrics in API Manager] relies on the Anypoint Monitoring agent. The agent is available by default for some Mule app deployment models. Other models require enablement of Anypoint Monitoring or installation of the agent. 
```

Comment:
```suggestion
Starting October 31, 2024, access to xref:api-manager::analytics-chart.adoc[Key Metrics in API Manager] requires Anypoint Monitoring to be enabled. Anypoint Monitoring is enabled by default for some Mule app deployment models. Other models require you to manually enable Anypoint Monitoring or to install the Anypoint Monitoring agent. 
```

Reply:
I don't think we can avoid talking about the agent.